### DISCUSSION TITLE
Feature suggestion: Password/Pin when app is opened

### DISCUSSION THREAD

**Discussion by torema-ed** [GitJournal/GitJournal#795] [Feature Requests]
Hi, thank you guys for the great work, I really like the innovative idea of using Git in this manner! I was the one who left the review about the password and thought it would be easier to submit an issue here instead of using Google Play Store review as a chat (Sorry, should have done this in the first place!)

Yes, you are right that I was suggesting to allow users to add a password or pin when app is opened. I think this would add an extra layer of security for people as a journal is a rather private thing.

**Comment by vHanda** [comment]
Note to self: https://pub.dev/packages/passcode_screen can be used to implement this quite easily.

**Comment by cyberbobcity** [comment]
I would like this feature as well!
